recipe/meta.yaml

```diff
@@ -7,23 +7,11 @@ package:
 
 source:
   fn: {{ name }}-{{ version }}.tar.gz
-  url: https://www.gap-system.org/pub/gap/gap48/tar.gz/gap4r8p10_2018_01_15-13_02.tar.gz
-  sha256: fc0dac32d196a825be5b4cab6a970aad58f1d16ca2870bd0381c94c85e7a9e5b
-  patches:
-    # Patches from https://github.com/sagemath/sage
-    # Set GAP_MEM
-    - patches/gap_shi.patch
-    # fix infinite loop in writeandcheck() when writing an error message fails.
-    - patches/writeandcheck.patch
-    # https://github.com/gap-system/gap/pull/448
-    - patches/448.patch
-    # https://github.com/gap-system/gap/pull/1188
-    - patches/path.patch
-    # we don't install some of the default packages
-    - patches/nodefaultpackages.patch
```

Comment:
This is still needed I think

Reply:
To consider Conda an alternative distribution for GAP, it will be desirable to have no differences between GAP installed from official distribution and from Conda. I am in favour of removing these patches - happy to discuss this further.

Reply:
@isuruf where can I see these patches? I can't find `patches` at https://github.com/sagemath/sage. As for ttps://github.com/gap-system/gap/pull/1188, it is included in GAP 4.9.

Reply:
Patches are at https://github.com/conda-forge/gap-feedstock/tree/master/recipe/patches

Reply:
Just to check, "This is still needed I think" refers only to the `patches/nodefaultpackages.patch` patch, isn't it? Regarding it, maybe you can provide some background - why did it appear in the first instance? Were some of these packages not available in conda? Do you know that there is a `-A` GAP command line option using which users can switch off loading of these packages when they start GAP?